api/api_public_blockchain.go

```diff
@@ -442,7 +462,7 @@ func RpcOutputBlock(b *types.Block, td *big.Int, inclTx bool, fullTx bool) (map[
 		"logsBloom":        head.Bloom,
 		"stateRoot":        head.Root,
 		"reward":           head.Rewardbase,
-		"blockscore":       (*hexutil.Big)(head.BlockScore),
+		"blockScore":       (*hexutil.Big)(head.BlockScore),
```

Comment:
I agree that `blockScore` is the corrent name because `Header` struct has the same json tag. However, this change may affect users or BApps parsing with the `blockscore` keyword. Could you investigate APIs which use `blockscore` or `blockScore` currently?

Reply:
I'll restore it and discuss it at the weekly meeting.
Thanks Aidan!